src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@englishman that's why, I don't think it's possible to have just one class `LengthOf`. We will need `LengthOfIterable`, `LenghtOfInput`, `LengthOfIterator`, etc.